src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classNames from 'classnames';
+import { EnumTodoFilter } from '../../types';
+
+type Props = {
+  filter: EnumTodoFilter;
+  onChange: (newFilter: EnumTodoFilter) => void;
+};
+
+export const TodoFilter: React.FC<Props> = React.memo(({
+  filter,
+  onChange,
+}) => (
+  <nav className="filter">
+    <a
+      href="#/"
+      className={classNames('filter__link', {
```

Comment:
you could've simplified it
```suggestion
      className={cn('filter__link', {
```